### PR TITLE
Release 1.104.0

### PR DESCRIPTION
**Pull request by fluiddot**
Release for Gutenberg Mobile 1.104.0

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/54458
- WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/19190
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/21576

## Extra PRs that Landed After the Release Was Cut

No extra PRs yet. 🎉

### RichText - Update logic for the placeholder text color
- **PR:** https://github.com/WordPress/gutenberg/pull/54259
- **Issue:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/6069

### feat: Insert from URL leverages a bottom sheet
- **PR:** https://github.com/WordPress/gutenberg/pull/54096
- **Issue:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/4936

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [x] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

File: bundle/android/strings.xml [hunk 1]
```diff
@@ -348,9 +348,6 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_slash_inserter_results" tools:ignore="UnusedResources">Slash inserter results</string>
     <string name="gutenberg_native_some_blocks_have_additional_settings_tap_the_settings_icon_on_the" tools:ignore="UnusedResources">Some blocks have additional settings. Tap the settings icon on the bottom right of the block to view more options.</string>
     <string name="gutenberg_native_start_writing" tools:ignore="UnusedResources">Start writing…</string>
-    <string name="gutenberg_native_take_a_photo" tools:ignore="UnusedResources">Take a Photo</string>
```
Comment (fluiddot) [inline bundle/android/strings.xml]: This string seems to be only used in the native version, so these changes should be reverted. Not sure why the script in charge of updating the localizations has removed them, I'd need to investigate it further 😞 .

Comment (fluiddot) [reply bundle/android/strings.xml]: Fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6204/commits/1b95e55629e4c9af7c0b17d14186222af0a596b9.

**Review by dcalhoun — APPROVED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -188,7 +188,7 @@
 				"eslint-plugin-jsdoc": "^46.4.6",
 				"eslint-plugin-jsx-a11y": "^6.5.1",
 				"eslint-plugin-playwright": "^0.15.3",
-				"eslint-plugin-prettier": "^3.3.0",
+				"eslint-plugin-prettier": "^5.0.0",
```
Comment (fluiddot) [inline package-lock.json]: This change is related to https://github.com/WordPress/gutenberg/pull/54539.

**Comment by fluiddot** [comment]
The failure in `Lint and Unit Tests ` will be solved in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6219.

**Error:**
```
TypeError: Error while loading rule 'prettier/prettier': context.getPhysicalFilename is not a function
Occurred while linting /workdir/__device-tests__/gutenberg-editor-functionality-test-1-visual.test.js
    at Object.create (/workdir/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js:140:40)
    at createRuleListeners (/workdir/node_modules/eslint/lib/linter/linter.js:746:21)
    at /workdir/node_modules/eslint/lib/linter/linter.js:916:31
    at Array.forEach (<anonymous>)
    at runRules (/workdir/node_modules/eslint/lib/linter/linter.js:861:34)
    at Linter._verifyWithoutProcessors (/workdir/node_modules/eslint/lib/linter/linter.js:1157:31)
    at Linter._verifyWithConfigArray (/workdir/node_modules/eslint/lib/linter/linter.js:1255:21)
    at Linter.verify (/workdir/node_modules/eslint/lib/linter/linter.js:1210:25)
    at Linter.verifyAndFix (/workdir/node_modules/eslint/lib/linter/linter.js:1400:29)
    at verifyText (/workdir/node_modules/eslint/lib/cli-engine/cli-engine.js:230:48)
```

**Pull request merged**